packages/eslint-plugin/lib/rules/no-nested-await.js

```diff
@@ -1,68 +1,68 @@
 /* global module, require */
 /**
  * @author Michael FIG
- * See LICENSE file in root direcotry for full license.
+ * See LICENSE file in root directory for full license.
  */
 
 'use strict';
 
-/**
- * Imports a specific module.
- *
- * @param {...string} moduleNames - module names to import.
- * @returns {object|null} The imported object, or null.
- */
-function safeRequire(...moduleNames) {
-  for (const moduleName of moduleNames) {
-    try {
-      // eslint-disable-next-line import/no-dynamic-require,global-require
-      return require(moduleName);
-    } catch (_err) {
-      // Ignore.
-    }
-  }
-  return null;
-}
-
-const CodePathAnalyzer = safeRequire(
-  'eslint/lib/linter/code-path-analysis/code-path-analyzer',
-  'eslint/lib/code-path-analysis/code-path-analyzer',
-);
+const isFunction = node => [
+  'ArrowFunctionExpression',
+  'FunctionExpression',
+  'FunctionDeclaration',
+].includes(node.type);
 
 module.exports = {
   meta: {
     docs: {
       description:
-        'prevent usage of `await` unless it is a function block statement or destructuring assignment',
+        `ensure the first \`await\` in an \`async\` function is non-nested so that it is clear when the synchronous portion of the function is finished`,
       category: 'Possible Errors',
       recommended: true,
       url:
-        'https://github.com/endojs/Jessie/blob/master/packages/eslint-plugin/lib/rules/no-nested-await.js',
+        'https://github.com/endojs/Jessie/blob/HEAD/packages/eslint-plugin/lib/rules/no-nested-await.js',
```

Comment:
I think our(undocumented) style is to use the branch name because it's a slightly better UX. See https://github.com/endojs/endo/pull/1473
```suggestion
        'https://github.com/endojs/Jessie/blob/main/packages/eslint-plugin/lib/rules/no-nested-await.js',
```